utils/types.ts

```diff
@@ -0,0 +1,14 @@
+import { ObjectID } from "mongodb";
+
+// Keep these in sync with the backend schema
+
+export interface User {
+  _id?: ObjectID;
+  email?: string;
+  password?: string;
```

Comment:
Can you remove the password field here? To let users quickly register for events, KKB prefers to not create accounts - so we won't need a password.